Oracle.md

```diff
@@ -95,17 +96,307 @@ unit: dollars
 precision: 0
 ```
 
-Alice and Bob create a DLC with "relevant" outcomes between $10000 and $10999.
-Below $10000 Alice gets everything, and above $10999 Bob gets everything.
-They use the `R0` and `R1` values to cover the cases where BTC/USD < $10000 using `D0=0` and `D1=0`.
-They use `R0`, `R1` and `R2` to cover the cases where BTC/USD > $10999 and <= $19999, using `D1=0`, `D1=1` and `D2 in [0-9]`.
-They use `R0` and `R1` to cover the cases where BTC/USD >= $20000 and < $99999, using `D0=0` and `D1 in [2,9]`.
-Finally, they use only `D0` to cover the cases where BTC/USD >= $100000 and <= $999999, using `D0 in [1-9]`.
+Alice and Bob create a DLC with "relevant" outcomes between $10,000 and $10,999.
+These relevant outcomes will be in $500 increments between $10,000 and $10,999.
```

Comment:
So after reading until the end, I think what you are describing is a binary option? So in that case I don't see why the relevant outcomes are between $10000 and $10999. And I still don't get the $500 increments part.

Reply:
The $500 increments is a random threshold I picked out of a hat. Would you prefer I do $250 increments? $100 increments? I choose $500 for the sake of brevity because it gets to be a lot to write out, but if the example isn't helpful to others if they don't understand what I am doing. Perhaps I need to reword the introduction to something like 

>Alice and Bob create a DLC with oscillating outcomes between $10,000 and $10,999.
These oscillation will occur in $500 increments between $10,000 and $10,999.
For this example, this means 1 oscillation will occur. 
If the BTC/USD price is < $10,000 Bob will receive everything.
If the BTC/USD price is >= $11,000 Alice will receive everything.